shadow.c

```diff
@@ -216,6 +216,7 @@ static void shadow_override_function(char *fname, size_t fname_len, int argno, i
 
     zend_string_release(fname_zs);
     zend_string_release(fname_full);
+    pefree(override, 1);
```

Comment:
I suppose we need to free these function overrides in `PHP_MSHUTDOWN_FUNCTION`

Reply:
that is very strange, I had no such failures in my build (see comments above with test output). I noticed the travis config is pinned to PHP 7.1.7 and I built with PHP 7.1.26, what version of PHP did you reproduce with?

Reply:
In your report I can see it's skipped for some reason:
```
SKIP Check chgrp() [tests/chgrp.phpt]
```
I tried it on 7.1.6 initially, now I've built it with 7.1.27 - same result, segfault:
```
$ cat tests/chgrp.out 
bool(true)
Segmentation fault (core dumped)

Termsig=11
```
```bash
$ php -v
PHP 7.1.27 (cli) (built: Mar 13 2019 17:22:43) ( NTS DEBUG )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2018 Zend Technologies
```

Reply:
sigh, I see what I did. thanks for catching! I'll do a PR without the pefree and see if I can at least successfully remove the warnings.